src/app/beer_garden/api/stomp/server.py

```diff
@@ -61,21 +76,24 @@ def on_error(self, headers, message):
         logger.warning("received an error:" + str(headers))
 
     def on_message(self, headers, message):
+        if headers.get("model_class") != "Operation":
+            return
```

Comment:
I removed the if statement since the try except should catch any error when an object received is not an operation, since we are assuming all messages received are operations.